src/framework/theme/styles/_theming.scss

```diff
@@ -101,14 +133,19 @@ $nb-themes-export: () !global;
 @mixin install-css-variables($theme-name, $variables) {
   .nb-theme-#{$theme-name} {
     @each $var, $value in $variables {
-      --#{$var}: $value;
+      --#{$var}: #{$value};
     }
   }
 }
 
-// TODO: we hide :host inside of it which is not obvious
-@mixin nb-install-component() {
+@mixin nb-install-component-new() {
```

Comment:
Maybe it's better to change `new` to `with-css-props-support`? Same for `old` and following `nb-install-global-new` and `nb-install-global-old` mixins.